portfolio/src/main/webapp/style.css

```diff
@@ -38,6 +38,7 @@ h2{
 h3{
     color:darkslategrey;
     font-family:Questrial;
+    style="margin:15px;"
```

Comment:
Should just be `margin: 15px;` the contents of the `style=` tag *is* CSS.

Reply:
This was a copy+paste error.